x-pack/plugins/security_solution/server/endpoint/ingest_integration.ts

```diff
@@ -31,85 +85,37 @@ export const getPackageConfigCreateCallback = (
     // follow the types/schema expected
     let updatedPackageConfig = newPackageConfig as NewPolicyData;
```

Comment:
Would `let updatedPackageConfig: NewPolicyData = newPackageConfig;` work here?

Reply:
@paul-tavares ^^

Reply:
No, we can't use it that way because our type enforces the `inputs[]` to have 1 member, but `inputs[]` in the `NewPackageConfig` can be empty.

TS error:
![image](https://user-images.githubusercontent.com/56442535/87815609-e5a67480-c833-11ea-8976-aa2b5a116b4e.png)